safe/test/utilities.py

```diff
@@ -55,7 +55,7 @@
     '..'))
 
 # Location of test data
-DATANAME = 'inasafe_data'
```

Comment:
Ah yes, we still have some test that use inasafe_data, but they are marked as expectedFailure